images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -77,13 +82,21 @@ $packageManagementList = @(
     (Get-HomebrewVersion),
     (Get-NPMVersion),
     (Get-YarnVersion),
-    (Get-NuGetVersion),
-    (Get-CondaVersion),
+    (Get-NuGetVersion)
+)
+
+if ($os.IsLessThanMonterey) {
+    $packageManagementList += @(
+        (Get-CondaVersion)
+    )
+}
+
+$packageManagementList += @(

```

Comment:
Why do we need these lines? Can we just put these 2 items to the `$packageManagementList` list above?

Reply:
Done. Thank you.